cmd/talisman/README.md

```diff
@@ -12,7 +14,14 @@ This will start a [Job](https://kubernetes.io/docs/concepts/workloads/controller
 kubectl logs -n kube-system -l job-name=talisman -f
 ```
 
-Run `./run_px_upgrade.sh --help` for more usage examples.
+### Restore scaled down shared applications
 
-Note: Currently, the only operation supported is `upgrade` in the `-operation` arguments of the talisman Job.
+During the upgrade, Portworx might scale down all shared volume PX appplications to 0 replicas.
+If the upgrade gets interuppeted in between by an unexpected failure, you can restore the shared volume PX applications back to their original replica count using below command.
```

Comment:
Could this be done automatically on failure? Could it be a flag to talisman to "autorecover-on-failure" , or more likely that should be the default, and we should have for debugging: `no-autorecover-on-failure`

Reply:
@lpabon  If talisman crashed and we re-run the same job again, it will continue doing what it was doing. It doesn't keep state but it's still idempotent.

This open is a failsafe when it keeps crashing due to a software bug (crash or unhandled target cluster conditions) and the `defer` to restore shared applications to their previous replica counts doesn't get invoked. So instead of end users having to manually do `kubectl edit deployment/statefulset` and edit the replica counts, this change provides as easier way to do that.

Reply:
Cool